site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Carvelizing Helm Chart"
+slug: carvelize-helm-chart
+date: 2022-02-16
+author: Rohit Aggarwal
+excerpt: "Use Carvel to author and consume helm chart "
+image: /img/logo.svg
+tags: ['carvel', 'helm', 'gitops']
+---
+# Carvelizing Helm Chart
+
+## Introduction
```

Comment:
taking a cue from other blogs, do we need this heading "Introduction". I think we can remove it